x-pack/plugins/gis/public/shared/layers/sources/ems_tms_source/ems_tms_source.test.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import {
+  EMSTMSSource,
+} from './ems_tms_source';
+
+describe('EMSTMSSource', () => {
+
+  it('should get attribution from markdown (tiles v2 legacy format)', async () => {
+
+    const emsTmsSource = new EMSTMSSource({
+      id: 'road_map'
+    }, {
+      id: 'road_map',
+      emsTmsServices: [
+        {
+          id: 'road_map',
+          attributionMarkdown: '[foobar](http://foobar.org)  | [foobaz](http://foobaz.org)'
+        }, {
+          id: 'satellite',
+          attributionMarkdown: '[foobar](foobaz)'
+        }
+      ]
+    });
+
+
+    const attributions = await emsTmsSource.getAttributions();
+
+    expect(attributions).toEqual([
+      {
+        label: 'foobar',
+        url: 'http://foobar.org'
+      }, {
+        label: 'foobaz',
```

Comment:
Why is the `satellite` attribution returned when the source only uses the `road_map` source?

Reply:
no it's not, but I realize now this tests is confusing in this way. I'll change the params for the satellite service instead.